<h1 style="text-align:center;">TÉCNICO EM CIÊNCIA DE DADOS</h1>

<h1 style="text-align:center;">Material Complementar</h1>

**Componente:** Matemática e Estatística para Ciência de Dados  
**Unidade Curricular:** Matemática Aplicada  
**Tema da Semana:** Operações Matriciais  
**Semana 2**  
**Aula 2:** Operações com matrizes


<h2 style="text-align:center;">Uma mudança no planejamento</h2>


## Finalidade

Notebook de conferência do **Colocando em Prática dos slides 49 e 50**. Use-o para verificar as saídas e apoiar a mediação sem antecipar os resultados aos estudantes.


## Dados da atividade

**Código Python**


In [1]:
T = [
    [117, 44],
    [157, 73]
]

R = [
    [8, 6],
    [5, 7]
]

C = [
    [10, 5],
    [20, 10]
]

print("T =", T)
print("R =", R)
print("C =", C)


T = [[117, 44], [157, 73]]
R = [[8, 6], [5, 7]]
C = [[10, 5], [20, 10]]


## Etapa 1 — Atualizar as participações

A adição é feita por entradas correspondentes.


In [2]:
P = [
    [T[0][0] + R[0][0], T[0][1] + R[0][1]],
    [T[1][0] + R[1][0], T[1][1] + R[1][1]]
]

P


[[125, 50], [162, 80]]

**Saída esperada:** `[[125, 50], [162, 80]]`


## Etapa 2 — Recalcular o orçamento

Aplique o procedimento linha × coluna.


In [3]:
o11_novo = P[0][0] * C[0][0] + P[0][1] * C[1][0]
o12_novo = P[0][0] * C[0][1] + P[0][1] * C[1][1]
o21_novo = P[1][0] * C[0][0] + P[1][1] * C[1][0]
o22_novo = P[1][0] * C[0][1] + P[1][1] * C[1][1]

O_atualizado = [
    [o11_novo, o12_novo],
    [o21_novo, o22_novo]
]

O_atualizado


[[2250, 1125], [3220, 1610]]

**Saída esperada:** `[[2250, 1125], [3220, 1610]]`


## Etapa 3 — Interpretação esperada

- **13 a 14 anos:** R$ 2.250 em materiais e R$ 1.125 em recursos digitais.
- **15 a 17 anos:** R$ 3.220 em materiais e R$ 1.610 em recursos digitais.
- A faixa de **15 a 17 anos** apresenta os maiores custos nas duas categorias.

A conclusão deve citar os valores da matriz como evidência.


## Ponto de atenção

Com listas Python, `T + R` concatena as duas listas; não realiza adição matricial. Se aparecer uma saída com quatro linhas, peça ao estudante que retome a soma por entradas correspondentes modelada no Desenvolvendo Juntos.
